pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -344,14 +345,23 @@ public static ElasticSearchParticipantDto getElasticSearchForGivenMatch(String i
         return esData;
     }
 
-    public static Map<String, Map<String, Object>> getFilteredDDPParticipantsFromES(@NonNull DDPInstance instance, @NonNull String filter) {
-        String index = instance.getParticipantIndexES();
-        if (StringUtils.isNotBlank(index)) {
+    public static Map<String, Map<String, Object>> getFilteredDDPParticipantsFromES(@NonNull DDPInstanceDto ddpInstanceDto,
```

Comment:
main idea of creating `ElasticSearchable` interface was to eliminate `ElasticSearchUtil` class, of course it can't be done immediately, but every time we need to add some new method which will work with ES, it is better to use `ElasticSearchable` interface and its subclass `ElasticSearch` or create new subclass, depends what task requires.